src/main/resources/db/changelog/changes/v1.0.4/sql/derived-tables/create-view-purchase-order-lines.sql

```diff
@@ -0,0 +1,56 @@
+CREATE OR REPLACE VIEW drv_purchase_order_line_details
+AS
+SELECT po.id,
+    po.jsonb ->> 'poLineNumber'::text AS purchase_order_line_number,
```

Comment:
The '->>' converts json into text, so I think that anywhere where we use a ->>, it's redundant to also have ::text at the end. We could probably remove the ::text just so that the view looks cleaner.